src/parsers/InputFieldDeclarationParser.ts

```diff
@@ -56,7 +81,60 @@ export class InputFieldDeclarationParser {
 
 	static templates: Template[] = [];
 
-	static parse(fullDeclaration: string): InputFieldDeclaration {
+	static parseDeclaration(
```

Comment:
maybe also make the function argument names consistent with the ones from `parseString`